pypeman/channels.py

```diff
@@ -608,13 +613,13 @@ def __init__(self, *args, basedir='', regex='.*', interval=1, binary_file=False,
         self.data = {}
         self.re = re.compile(self.regex)
         self.binary_file = binary_file
-
+        self.real_extensions = real_extensions  # list of extensions for exemple: [".csv", ".CSV"]
         # Set mtime for all existing matching files
         if os.path.exists(self.basedir):
```

Comment:
you can even use pathlib for this and `listdir` I think

Reply:
or you use 
```
for filepath in Path(self.basedir).glob("*"):
```

Reply:
doen